website/www/site/data/en/join_beam.yaml

```
@@ -0,0 +1,29 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- list:
+    - { title: Mailing Lists, description: 'Join the official Apache Beam user and developer mailing lists. Learn how <a href="/community/contact-us/">here</a>' }
+    - { title: Slack Channel, description: 'For real-time communication, join us on the <a target="_blank" href="https://app.slack.com/client/T4S1WH2J3/C9H0YNP3P">#beam</a> channel in the ASF workspace.' }
+    - { title: Twitter account, description: Follow  '<a target="_blank" href="https://twitter.com/apachebeam?lang=en">@ApacheBeam</a> on Twitter for the most recent news on the project and community!' }
```

Fix my typo. This broke the PreCommit. Sorry!

```suggestion
    - { title: Twitter account, description: 'Follow <a target="_blank" href="https://twitter.com/apachebeam?lang=en">@ApacheBeam</a> on Twitter for the most recent news on the project and community!' }
```